### PR TITLE
Cloud IDE Day Event Landing Page #96

### LINKED ISSUE #96: Cloud IDE Day Event Landing Page
The ECD Tools WG will be putting on Cloud IDE Day, a new event on May 19. They would like an event landing page created off of the ecdtools.eclipse.org page. 

Here is the website content form: https://docs.google.com/document/d/1tV37AISjLhP1wsUpyEKAFAVIgzTfjU98Z-_SxMgPfno/edit#

Here are the images for the program committee 
[Program Committee.zip](https://github.com/EclipseFdn/ecdtools.eclipse.org/files/6091586/Program.Committee.zip)

Here is the ECD Tools SVG Logo for the 'Organized by:' section in the papercall 
[Untitled design (4).zip](https://github.com/EclipseFdn/ecdtools.eclipse.org/files/6091605/Untitled.design.4.zip)

Let me know if anything else is needed

### PR DESCRIPTION
**Pull request by flora8984461**
Fix #96 

- committee
- call for paper
- organized by

### CODE REVIEW

File: webpack.mix.js [hunk 1]
```diff
@@ -23,14 +23,15 @@ mix.babel([
...
     './node_modules/jquery-match-height/dist/jquery.matchHeight-min.js',
+    './node_modules/jquery-eclipsefdn-api/dist/jquery.eclipsefdn-api.min.js',
     './node_modules/feather-icons/dist/feather.min.js',
     './node_modules/cookieconsent/src/cookieconsent.js',
-    './node_modules/jquery-eclipsefdn-api/dist/jquery.eclipsefdn-api.min.js',
+    './node_modules/owl.carousel/dist/owl.carousel.min.js',
     './node_modules/eclipsefdn-solstice-assets/js/solstice.cookieconsent.js',
     './node_modules/eclipsefdn-solstice-assets/js/eclipsefdn.videos.js',
     './node_modules/eclipsefdn-solstice-assets/js/solstice.cookies.js',
-    './node_modules/eclipsefdn-solstice-assets/js/eclipsefdn.render-rss-feeds.js',
```
Comment (chrisguindon) [inline webpack.mix.js:32]: Was it intentional to remove this?

Comment (flora8984461) [reply webpack.mix.js:32]: Ah, unintentional, let me add it back

File: config.toml [hunk 2]
```diff
@@ -8,6 +8,7 @@ disableKinds = []
 themesDir = "node_modules/"
 enableRobotsTXT = true
 pluralizeListTitles = false
+buildFuture = true # A new event website will always be in the future
```
Comment (chrisguindon) [inline config.toml:11]: This should not be there.

Comment (chrisguindon) [reply config.toml:11]: See: https://stackoverflow.com/questions/59655470/hugo-date-vs-publishdate#:~:text=Date%20%2D%20the%20date%20associated%20with,the%20content%20was%20last%20modified

File: webpack.mix.js (hunk 1 above)
Comment (flora8984461) [reply webpack.mix.js:32]: added back

File: layouts/shortcodes/events/sponsors.html [hunk 3]
```diff
@@ -0,0 +1,79 @@
+<!--
```
Comment (flora8984461) [inline layouts/shortcodes/events/sponsors.html:1]: This is for https://github.com/EclipseFdn/ecdtools.eclipse.org/issues/96#issuecomment-797022680 "4: Under the 'Organized By' section, please add the following phrase above the ECD Tools logo:" 

I temporarily choose to use override, while I am not sure if I need to add this feature into the base theme. 🤔 

What do you think?

Comment (flora8984461) [reply layouts/shortcodes/events/sponsors.html:1]: I only added a 

```
  <div class="eclipsefdn-registration-body">
    {{ .Inner | markdownify }}
  </div>
```

File: config.toml (hunk 2 above)
Comment (flora8984461) [reply config.toml]: removed

File: layouts/shortcodes/events/sponsors.html (hunk 3 above)
Comment (chrisguindon) [reply layouts/shortcodes/events/sponsors.html:1]: We can add it to the base theme but please make sure to only include the div if .Inner is not empty

**Pull request merged**